cmake/modules/RootBuildOptions.cmake

```diff
@@ -409,11 +409,6 @@ foreach(opt minuit2_omp minuit2_mpi)
   endif()
 endforeach()
 
-#---Replaced options--------------------------------------------------------------------------
-if(python)
```

Comment:
Beside saving 3 lines of code, what is the advantage of removing this warning?  Users might very well 'upgrade' from a version that had the `python` option directly to `v6.38` without first testing the intermediaries.

Reply:
Please see the commit message (that's why I write them...):
> It was modified in commit 5410bb44676ee7bd87581a073b6644513f74aac9, but that never worked because it was also added to the list of removed options, which will issue a `FATAL_ERROR` if enabled.

In other words, such user would never see the warning anyway.

Reply:
Fair enough.  Thanks.